### DISCUSSION TITLE
Show outline only during keyboard navigation

### DISCUSSION THREAD

**Discussion by ankurdeka** [radix-ui/primitives#1857] [Help]
https://user-images.githubusercontent.com/71803702/209469827-35532049-5536-47a0-8fbb-b351e099d56e.mp4

How to show the outline only when user is using keyboard?
`:focus-visible` doesn't work

**Comment by ankurdeka** [comment]
https://user-images.githubusercontent.com/71803702/209472294-b4ddc66b-8ade-4936-bbd8-6eff2bb482de.mp4

**One weird behavior**
On a fresh page dropdown items show outline when the mouse is hovered but after clicking the input element the dropdown items doesn't show any outline. 

Is this a bug or am I doing something wrong?

repo : [https://github.com/ankurdeka/twitch-clone-frontend](https://github.com/ankurdeka/twitch-clone-frontend)

**Reply by benoitgrelard** [reply]
See this: https://github.com/radix-ui/primitives/issues/1803#issuecomment-1400023626
